clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/LeaderFollowerStoreIngestionTask.java

```diff
@@ -2259,23 +2259,20 @@ protected void recordHeartbeatReceived(
       return;
     }
 
-    // Separate incremental push pubsub entries has the same pubsub url but different cluster id, which creates
```

Comment:
Is this not required for IncPush topic anymore? OR is it the case that URL trickled down from `StorePartitionDataReceiver` is already resolved?

Reply:
Good question.
I read the code a little bit and I am guessing the Kafka url for inc push RT topic is having `_sep` suffix, so does `getKafkaClusterUrlToAliasMap` contain the right alias for such special urls?

Reply:
So for original RT this does not change anything; For separate RT, it will find the correct cluster ID in the record heartbeat method. 
We will only register metric and sensor that these new regions when version level config is enabled.

Reply:
If I understand your comment correctly, `kafkaClusterUrlToAliasMap` does contain the alias for the special urls for inc push RT topics, right?

Reply:
yes it does, if we deployed the config change. So far it is deployed and verified in two test clusters.